cmd/influxd/config.go

```diff
@@ -52,13 +52,8 @@ type Config struct {
 		ReadTimeout Duration `toml:"read-timeout"`
 	} `toml:"api"`
 
+	Graphite     []Graphite `toml:"graphite"`
```

Comment:
I feel like this should be plural:

```
Graphites     []Graphite `toml:"graphite"` 
```

Otherwise it feels kind of weird in the code since it doesn't look like a collection.